client/app/helpers/to-locale-string.js

```diff
@@ -0,0 +1,5 @@
+import { helper } from '@ember/component/helper';
+
+export default helper(function toLocaleString(number) {
+  return number.toLocaleString();
```

Comment:
aw okay so this makes it so that it match the language-sensitive? cool! Haven't seen this before